comps/cores/proto/docarray.py

```diff
@@ -62,6 +62,10 @@ class EmbedDoc1024(BaseDoc):
     text: str
     embedding: conlist(float, min_length=1024, max_length=1024)
 
+class EmbedDoc512(BaseDoc):
```

Comment:
Is this length 512 a must? We actually plan to remove all the hard limitation of embedding length in this [PR](https://github.com/opea-project/GenAIComps/pull/347). Do you think that fit this case?

Reply:
The embeddings we will return from our current embedding model are 512, but there isn't a need to have this as a strict constraint in the pydandic model. I've removed the constraints for now and added a `TODO` comment to update to the new `EmbedDoc` whenever that is merged (depending if that comes before or after this PR is merged.

Reply:
Ah @Spycsh, I didn't realize that other PR was already merged. I updated our embedding component accordingly.